### DISCUSSION TITLE
Question downloading data from qrz.com

### DISCUSSION THREAD

**Discussion by adamigom** [magicbug/Cloudlog#1731] [Ideas / Feature Requests]
I have a question if there are plans to launch an option to download data from qrz.com. In the offline app on android it works ok but already on the website there is no this function. Request for information.

I am most interested in information name qth and gird.
![Screenshot_2022-10-30-23-58-26-51_40deb401b9ffe8e1df2f1cc5ba480b12](https://user-images.githubusercontent.com/117047000/198905949-ec96e256-2c5b-482d-a53a-cf281bb95734.jpg)

**Comment by adamigom** [comment]
Do you need to make any additional settings ?

**Comment by phl0** [comment]
You have to configure the qrz.com credentials. Not sure if a (paid) subscription is required.

**Comment by adamigom** [comment]
![IMG_20221031_101453](https://user-images.githubusercontent.com/117047000/198973724-a38ac1cb-8b28-4083-b451-7974bcf6feb6.jpg)

I have my data completed, I do not have a paid subscription but in other applications it always downloads my basic data Name qth.

As for hamqth it downloads all my data in other apps.

Does cloudlog have options to download data from qrz.com and hamqth.com ?


By the way what do I need to set in the options so that the known location changes me to gird and completes in the qso window ?

**Reply by phl0** [reply]
This is the wrong place. You need to set your username and password in `application/config/config.php`:

```
$config['qrz_username'] = "M0ABC";
$config['qrz_password'] = "ThisIsAverySecurePassword";
$config['use_fullname'] = false;
```

**Comment by AndreasK79** [comment]
https://github.com/magicbug/Cloudlog/wiki/Callsign-Lookup

**Comment by adamigom** [comment]
Thanks it was helpful.
